internal/daemon/api_services_test.go

```diff
@@ -321,15 +320,3 @@ func (s *apiSuite) TestServicesReplan(c *C) {
 	c.Check(tasks[0].Summary(), Equals, `Start service "test1"`)
 	c.Check(tasks[1].Summary(), Equals, `Start service "test2"`)
 }
-
-type wrappedServiceManager struct {
```

Comment:
This wasn't being used anywhere, deleting.

Reply:
Thanks. I don't even know why this was here.